### ISSUE TITLE
setExternalUserId should accept an additional authentication parameter

### ISSUE THREAD

**Issue by statico** [OneSignal/react-onesignal#33]
In order to use OneSignal identity verification, `setExternalUserId` should accept a second parameter: https://documentation.onesignal.com/docs/identity-verification

This can be worked around currently by doing:

```typescript
    const instance = OneSignal.getOneSignalInstance()
    instance.setExternalUserId(externalUserId, externalUserIdAuthHash)
```

**Comment by MatthewQuek** [comment]
same thing for setEmail

**Comment by rgomezp** [comment]
Howdy,
This is now available in the 2.0 Beta release.

Enjoy!
